acre.yaml

```diff
@@ -558,7 +569,19 @@ validation:
     selfsigned: (( |ca,url|-> [url == "self-signed" ? valid( ca.crt ) -and valid( ca.key ) :true, "valid", "if 'landscape.cert-manager.server.url' is set to 'self-signed', you should either specify a CA with key ('server.ca.crt' and 'server.ca.key' set), or no CA at all ('server.ca' node not present)"] ))
     acme: (( |ca,url|-> [url != "self-signed" ? valid( ca.crt ) -and ( ! defined( ca.key ) ) :true, "if 'landscape.cert-manager.server.url' is not set to 'self-signed', you should either specify a CA without key ('server.ca.crt' set, no 'server.ca.key'), or the 'server.ca' node should not be present at all"] ))
     email: (( |ca,id|-> [valid( ca.email ) -or valid( id.users[0].email ), "either 'landscape.cert-manager.email' or 'landscape.identity.users[0].email' needs to be set"] ))
-
+  extensionsValuesOverwrite_validator:
+    - and
```

Comment:
This validation is currently kind of useless - you check that _all_ extensions are listed, but since you add entries for all extensions above, this will always be true.

Reply:
Instead, we could validate that no entry is specified that doesn't match to one of the extensions (I think this is what you actually wanted to do). I think we don't need this entry for that, I'll describe what it could look like below.